gas_tank/gas_tank_problem.ipynb

```diff
@@ -186,6 +186,12 @@
     }
```

Comment:
<p>Don't repeat information from cell 1.</p><p>* Remove data from cell 1</p><p>* Write a docstring for `purchase_plan` that describes each of the inputs</p>

 _Reply via <a href='https://app.reviewnb.com/kiwidamien/job_questions/pull/1/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>